SPECS/tpm2-tss/tpm2-tss.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "tpm2-tss-2.4.6.tar.gz": "20e6da532a7ef90c8e50cca51f276053ec505eee0167c18e2b07c1e747118b58"
```

Comment:
This is a significant version bump for 1.0, is there any  hope of backporting the patch for this?

Reply:
Good point. Looked into it and seems like the patch can be applied to the current source code. Have re-implemented the patch.